gadget/ondisk.go

```diff
@@ -371,35 +371,3 @@ func filesystemInfoForPartition(node string) (blk lsblkBlockDevice, err error) {
 func toUpperUUID(bd *lsblkBlockDevice) {
 	bd.UUID = strings.ToUpper(bd.UUID)
 }
-
-func listBlockDevices(devType string) ([]lsblkBlockDevice, error) {
-	output, err := exec.Command("lsblk", "--json").CombinedOutput()
```

Comment:
Much happier that we're doing what lsblkd did, but just the bits that we really require.